src/main/java/com/stripe/param/TokenCreateParams.java

```diff
@@ -0,0 +1,2527 @@
+// Generated by com.stripe.generator.entity.SdkBuilder
+
+package com.stripe.param;
+
+import com.google.gson.annotations.SerializedName;
+import com.stripe.net.ApiRequestParams;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import lombok.Getter;
+
+@Getter
+public class TokenCreateParams extends ApiRequestParams {
+  @SerializedName("account")
+  Account account;
+
+  @SerializedName("amount")
+  Long amount;
```

Comment:
This should not be in this API. Please make sure to remove/document in pay-server so that it can't leak again. This is a really old and sensitive parameter that lets you do card testing really easily :(

Reply:
This and the token fields below are addressed